src/prism.c

```diff
@@ -16757,6 +16773,16 @@ parse_expression(pm_parser_t *parser, pm_binding_power_t binding_power, pm_diagn
         return node;
     }
 
+    // The statements BEGIN { ... }, END { ... }, alias ..., and undef ... are statement.
```

Comment:
Because this is such a hot path through the parser, can you make this into a single switch statement and combine it with the `PM_MISSING_NODE` check above? The compiler _probably_ makes it equivalent, but I'd like to be explicit about it.